docbot/action.go

```diff
@@ -47,11 +47,16 @@ func NewActionWithClient(ctx context.Context, ac *ActionConfig, client *github.C
 	}
 }
 
+const (
+	openedActionType = "opened"
+	editedActionType = "edited"
+)
+
 func (a *Action) Run(prNumber int, actionType string) error {
 	a.prNumber = prNumber
 
 	switch actionType {
-	case "opened", "edited", "labeled", "unlabeled":
+	case openedActionType, editedActionType:
```

Comment:
I think we should handle labeled/unlabeled action here to undo unintended editions?

For example, if only "doc-not-needed" box is checked and a committer check "doc" label, we should remove it.